src/tests/ftest/io/eight_servers_mpiio.yaml

```diff
@@ -40,30 +40,31 @@ ior:
             np: 64
         np_128:
             np: 128
-    iteration:
-        iter: 1
     iorflags:
         ssf:
             F: "-w -r"
         fpp:
             F: "-w -r -F"
+    api: MPIIO
+    test_file: daos:testFile
+    repetitions: 1
```

Comment:
Could you make the repetitions as 3, it seems like it was wrong originally.

Reply:
Sure